API.yaml

```diff
@@ -1395,6 +1397,18 @@ paths:
                       type: array
                       items:
                         type: string
+  /ping:
```

Comment:
status?

Reply:
@tahini @MatthewKhouzam Or something like `/health` with custom status code like up, down, unknown, etc. Spring Actuator and Micronaut management are using `/health` to monitor the microservices

Reply:
I like!

Reply:
To expand on my previous comment.

https://www.callicoder.com/spring-boot-actuator/

maybe we should have: 
* a `/health` endpoint for basics <-- ping goes here
* a `/metrics` endpoint for measurable metrics
* a `/logger` endpoint to access the logs

thoughts?